Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -473,8 +474,35 @@ internal AudioClip FindAudio(int id)
         #endregion
 
         #region ReceiveScene
-        internal void BeforeUpdateScene() {
+
+        internal void BeforeUpdateScene(FenceMessage? mes = null) {
             onSceneUpdateBegin?.Invoke();
+
+            if (mes.HasValue && currentSessionId != mes.Value.SessionId) {
```

Comment:
Will this pop-up on the first time usage? Might be good to also check the child count of the server

Reply:
Fixed